modules/ROOT/pages/deployment/container/orchestration/orchestration.adoc

```diff
@@ -499,6 +499,75 @@ minikube kubectl -- apply -f generic-secrets.yaml
 
 . To apply secrets via your own `values.yaml`, add the content at `extraResources`. Proper yaml formatting is necessary.
 
+==== Define Generic Configs
+
+Infinite Scale requires some generic configuration to work. It was decided not to create them automatically, because Helm does not support _one-off_ generation of configuration out of the box.
```

Comment:
```suggestion
Infinite Scale requires some generic configuration to work. It was decided not to create them automatically, because Helm does not support _on-off_ generation of configuration out of the box.
```